controllers/controllers/cluster_controller.go

```diff
@@ -72,6 +83,19 @@ func (r *ClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request) (_
 		return ctrl.Result{}, nil
 	}
 
+	// Fetch the VsphereDatacenter object
+	dc := &anywherev1.VSphereDatacenterConfig{}
```

Comment:
nit: maybe name the var `datacenterconfig` instead of short? i did the same thing in original controller and found out later it's a bit hard to follow. (vmc, vdc, etc lol)

Reply:
done